### PR TITLE
seqtk: import from homebrew/science

### PR DESCRIPTION
**Pull request by sjackman**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/seqtk.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Seqtk < Formula
+  desc "Toolkit for processing sequences in FASTA/Q formats"
+  homepage "https://github.com/lh3/seqtk"
+  url "https://github.com/lh3/seqtk/archive/v1.2.tar.gz"
+  sha256 "bd53316645ab10f0aaba59e1e72c28442ee4c9c37fddaacce5e24757eff78d7b"
+  head "https://github.com/lh3/seqtk.git"
+
+  def install
+    system "make"
+    bin.install "seqtk"
```
Comment (ilovezfs) [inline Formula/seqtk.rb]: Have you requested a `make install` target?

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/seqtk.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/seqtk.rb]: Done: https://github.com/lh3/seqtk/issues/103

File: Formula/seqtk.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Seqtk < Formula
+  desc "Toolkit for processing sequences in FASTA/Q formats"
+  homepage "https://github.com/lh3/seqtk"
+  url "https://github.com/lh3/seqtk/archive/v1.2.tar.gz"
+  sha256 "bd53316645ab10f0aaba59e1e72c28442ee4c9c37fddaacce5e24757eff78d7b"
+  head "https://github.com/lh3/seqtk.git"
```
Comment (ilovezfs) [inline Formula/seqtk.rb]: let's remove head

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/seqtk.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/seqtk.rb]: needs to be merged upstream.

**Pull request closed without merging**